lib/models/ergometer.dart

```diff
@@ -25,7 +25,8 @@ class Ergometer {
   /// This is intended only for internal use by [ErgBleManager.startErgScan].
   /// Consider this method a private API that is subject to unannounced breaking
   /// changes. There are likely much better methods to use for whatever you are trying to do.
-  Ergometer(this._peripheral);
+  Ergometer(this._peripheral, {FlutterReactiveBle? bleClient})
```

Comment:
This change was first intended for testing but we should probably pass the FlutterReactiveBle instance from ErgBleManager
```suggestion
  Ergometer(this._peripheral, {required FlutterReactiveBle bleClient})
```
This way we have a single FlutterReactiveBle instance in memory and it should help with the testing.